service/src/services/MeasureService.ts

```diff
@@ -40,8 +41,16 @@ export class MeasureService implements Service<fhir4.Measure> {
     logger.debug(`Request Query: ${JSON.stringify(query, null, 2)}`);
     const parsedQuery = parseRequestSchema(query, MeasureSearchArgs);
     const mongoQuery = getMongoQueryFromRequest(parsedQuery);
-    const entries = await findResourcesWithQuery<fhir4.Measure>(mongoQuery, 'Measure');
-    return createSearchsetBundle(entries);
+
+    // if the _summary parameter with a value of count is included, then
+    // return a searchset bundle that excludes the entries
+    if (parsedQuery._summary && parsedQuery._summary === 'count') {
+      const count = await findResourceCountWithQuery(mongoQuery, 'Measure');
+      return createSummarySearchsetBundle<fhir4.Measure>(count);
+    } else {
```

Comment:
It looks like we default to a normal searchset bundle if the query includes the `_summary` parameter but the value isn't `count`. Do we want to throw an unsupported error for that case instead?